### PR TITLE
Misc cleanups needed for chain deploy

### PR DESCRIPTION
**Pull request by michaelfig**
See the individual commit messages.  These were too small to land in separate PRs.

### CODE REVIEW

File: packages/agoric-cli/lib/deploy.js [hunk 1]
```diff
@@ -63,11 +63,6 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
     .filter(dep => dep && !provide.includes(dep))
     .sort();
 
-  if (args.length === 0 && !provide.length) {
```
Comment (dckc) [inline packages/agoric-cli/lib/deploy.js:66]: how does this allow anything additional? it seems to deny something.

File: packages/agoric-cli/lib/start.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ import {
 
 import { makePspawn, getSDKBinaries } from './helpers.js';
 
-const PROVISION_COINS = `100000000${STAKING_DENOM},50000000000${CENTRAL_DENOM},100provisionpass,100sendpacketpass`;
+const PROVISION_COINS = `100000000${STAKING_DENOM},500000000000000${CENTRAL_DENOM},100provisionpass,100sendpacketpass`;
```
Comment (dckc) [inline packages/agoric-cli/lib/start.js:15]: this affects the faucet that testnet validators use, right? I was going to ask you about this.

File: packages/agoric-cli/lib/deploy.js (hunk 1 above)
Comment (dckc) [reply packages/agoric-cli/lib/deploy.js:66]: ah. I read it backwards. this is removing a constraint

**Review by dckc — APPROVED** [review]
Yea and verily!

File: packages/agoric-cli/lib/start.js (hunk 2 above)
Comment (michaelfig) [reply packages/agoric-cli/lib/start.js:15]: Actually, just for local chain use.  On-chain is configured in `packages/deployment`, which we use for testnet genesis generation.

**Pull request merged**